pkg/storage-init/storage-init.sh

```diff
@@ -23,6 +23,12 @@ else
     echo "$(date -Ins -u) No separate $CONFIGDIR partition"
 fi
 
+
+STORAGE_OVERRIDE_CONFIG="${CONFIGDIR}"/storage.cfg
+if [ -f "${STORAGE_OVERRIDE_CONFIG}" ]; then
+    source "${STORAGE_OVERRIDE_CONFIG}"
```

Comment:
Do we really need to run source command here? It may come with unexpected affect to existing flow. Can we use some subset of options inside and parse them?

In general, I thing better to have some document, but no file included in conf directory. If we add file there we will have additional questions about how to use it, but for now usage is quite limited.

Reply:
Good point. I added same primitive parsing. Now only expected variables are extracted. I'll update documentation as well, once we reached an agreement on how to use these overrides cfg files